### PR TITLE
More What's New for EF Core 6.0

### PR DESCRIPTION
**Pull request by ajcvickers**

### CODE REVIEW

**Comment by ajcvickers** [comment]
@dotnet/efteam Updated to contain most of the big features. Still quite a few smaller features to do. Would be great to get this pushed live later today so that people can refer to it when trying out RC1. Doesn't have to be perfect immediately.

**Comment by bricelam** [comment]
Bundles section: LGTM

File: entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md [hunk 1]
```diff
@@ -291,25 +291,360 @@ Historical data for Rainbow Dash:
...
+Because of these limitations, you should only use compiled models if your EF Core startup time is too slow. Compiling small models is typically not worth it.
+
+If supporting any of these features is critical to your success, then please vote for the appropriate issues linked above.
+
+### Benchmarks
+
+> [!TIP]
+> You can try compiling a large model and running a benchmark on it by [downloading the sample code from GitHub](https://github.com/dotnet/EntityFramework.Docs/tree/main/samples/core/Miscellaneous/CompiledModels).
+
+The model in the GitHub repo referenced above contains 449 entity types, 6390 properties, and 720 relationships. This is a moderately large model. Using [BenchmarkDotNet](https://www.nuget.org/packages/BenchmarkDotNet) to measure, the average time to first query is 1.02 seconds on a reasonably powerful laptop. Using compiled models brings this down to 117 milliseconds on the same hardware. An improvement of 10 to 12 percent like this stays relatively constant as the model size increases.
```
Comment (AndriySvyryd) [inline entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md:626]: It's an improvement of 8 to 9 times

**Review by AndriySvyryd — APPROVED** [review]
I would really prefer that the binary database file is not checked in.

File: entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md [hunk 2]
```diff
@@ -291,25 +291,360 @@ Historical data for Rainbow Dash:
...
+> See [Announcing Entity Framework Core 6.0 Preview 6: Configure Conventions](https://devblogs.microsoft.com/dotnet/announcing-entity-framework-core-6-0-preview-6-configure-conventions/) on the .NET Blog for more discussion and examples of pre-convention model configuration.
 
 ## Compiled models
 
 GitHub Issue: [#1906](https://github.com/dotnet/efcore/issues/1906).
 
-// TODO
+Compiled models can improve EF Core startup time for applications with large models. A large model typically means 100s to 1000s of entity types and relationships.
+
+Startup time typically means the time to perform the first operation on a DbContext when that DbContext type is used for the first time in the application. Note that just creating a DbContext instance does not cause the EF model to be initialized.Instead, typical first operations that cause the model to be initialized include calling `DbContext.Add` or executing the first query.
```
Comment (roji) [inline entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md:579]: (lots of "typically" around :))

```suggestion
Startup time means the time to perform the first operation on a DbContext when that DbContext type is used for the first time in the application. Note that just creating a DbContext instance does not cause the EF model to be initialized. Instead, typical first operations that cause the model to be initialized include calling `DbContext.Add` or executing the first query.
```

File: entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md (hunk 1 above)
Comment (roji) [reply entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md:626]: One could even say "almost tenfold improvements" :)

File: entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md [hunk 3]
```diff
@@ -291,25 +291,360 @@ Historical data for Rainbow Dash:
...
 ## Improved performance on TechEmpower Fortunes
 
 GitHub Issue: [#23611](https://github.com/dotnet/efcore/issues/23611).
 
-// TODO
+We made significant improvements to query performance for EF Core 6.0. Specifically:
+
+* EF Core 6.0 performance is now 70% faster on the industry-standard TechEmpower Fortunes benchmark, compared to 5.0.
+  * This is the full-stack perf improvement, including improvements in the benchmark code, the .NET runtime, etc.
+* EF Core 6.0 itself is 31% faster executing queries.
```
Comment (roji) [inline entity-framework/core/what-is-new/ef-core-6.0/whatsnew.md:641]:
```suggestion
* EF Core 6.0 itself is 31% faster executing (untracked) queries.
```

**Review by roji — APPROVED** [review]

**Pull request merged**